app/Config/core.php

```diff
@@ -254,20 +254,20 @@
  *
  * To use, prefix your JavaScript link URLs with '/cjs/' instead of '/js/' or use JsHelper::link().
  */
-	//Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');
+    //Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');" but found "//Configure::write('Asset.filter.js', 'custom_javascript_output_filter.php');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment